pkg/appgw/appgw_test.go

```diff
@@ -379,6 +381,11 @@ var _ = Describe("Tests `appgw.ConfigBuilder`", func() {
 
 	BeforeEach(func() {
 		stopChannel = make(chan struct{})
+		appGwIdentifier = Identifier{
```

Comment:
I don't think we expect mutations on this - so we could pull it out of `BeforeEach` to speed up tests